pkg/storaged/vdo-details.jsx

```diff
@@ -256,16 +256,18 @@
 
         const header = (
             <Card>
-                <CardHeader>
-                    <CardTitle><Text component={TextVariants.h2}>{cockpit.format(_("VDO device $0"), vdo.name)}</Text></CardTitle>
-                    <CardActions>
+                <CardHeader actions={{
+                    actions: <>
                         { block
                             ? <StorageButton onClick={stop}>{_("Stop")}</StorageButton>
                             : <StorageButton onClick={vdo.start}>{_("Start")}</StorageButton>
                         }
                         { "\n" }
                         <StorageButton kind="danger" onClick={delete_}>{_("Delete")}</StorageButton>
-                    </CardActions>
+                    </>,
+                }}>
+                    <CardTitle><Text component={TextVariants.h2}>{cockpit.format(_("VDO device $0"), vdo.name)}</Text></CardTitle>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-112552-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)